openlibrary/plugins/openlibrary/js/goodreads_import.js

```diff
@@ -52,76 +48,108 @@ export function initGoodreadsImport() {
         $('input.import-submit').addClass('hidden');
         $('th.import-status').removeClass('hidden');
         $('th.status-reason').removeClass('hidden');
+        const shelves = { read: 3, 'currently-reading': 2, 'to-read': 1 };
         count = 0;
         prevPromise = Promise.resolve();
         $('input.add-book').each(function () {
             var input = $(this),
                 checked = input.prop('checked');
             var value = JSON.parse(input.val().replace(/'/g, '"'));
             var shelf = value['Exclusive Shelf'];
-            const shelves = {read: 3, 'currently-reading': 2,  'to-read': 1};
             var shelf_id = 0;
-            if (shelves[shelf]){
+            var has_failure = function () {
```

Comment:
```suggestion
            const hasFailure = function () {
```
Camel case should be used for Javascript variable names.